configure.ac

```diff
@@ -219,10 +219,11 @@ AS_IF([test "x$enable_java" != "xno"], [
 ]) dnl test enable_java
 
 dnl Test is ant is available since we now use it for the build system
+dnl BC - TODO - Add this to the enable Java section and uncomment below
 AC_PATH_PROG([ANT], [ant], [no])
-if test "x$ANT" = "xno"; then
-   AC_MSG_ERROR([Apache Ant is required but was not found. Install Ant and try again.])
-fi
+dnl if test "x$ANT" = "xno"; then
+dnl    AC_MSG_ERROR([Apache Ant is required but was not found. Install Ant and try again.])
+dnl fi
 AC_SUBST(ANT)
```

Comment:
We can remove this, but then one of the `make clean` targets is going to fail, because many of the things made by `ant` can only be cleaned by ant. So we will need to conditionalize the `make clean` as well.

Reply:
And this should be moved into the Java section then, which I will do.